### PR TITLE
[Core] Extend ocean new cli command

### PR DESCRIPTION
**Pull request by yaelibarg**
# Description

fixed the flow of the ocean new command since it was not working properly by getting all data needed from the user and filling it out for him, and added debug file 

## Type of change

Please leave one option from the following and delete the rest:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Documentation (added/updated documentation)

<h4> All tests should be run against the port production environment(using a testing org). </h4>

### Core testing checklist

- [ ] Integration able to create all default resources from scratch
- [ ] Resync finishes successfully
- [ ] Resync able to create entities
- [ ] Resync able to update entities
- [ ] Resync able to detect and delete entities
- [ ] Scheduled resync able to abort existing resync and start a new one
- [ ] Tested with at least 2 integrations from scratch
- [ ] Tested with Kafka and Polling event listeners
- [ ] Tested deletion of entities that don't pass the selector

### Integration testing checklist

- [ ] Integration able to create all default resources from scratch
- [ ] Resync able to create entities
- [ ] Resync able to update entities
- [ ] Resync able to detect and delete entities
- [ ] Resync finishes successfully
- [ ] If new resource kind is added or updated in the integration, add example raw data, mapping and expected result to the `examples` folder in the integration directory.
- [ ] If resource kind is updated, run the integration with the example data and check if the expected result is achieved
- [ ] If new resource kind is added or updated, validate that live-events for that resource are working as expected
- [ ] Docs PR link [here](#)

### Preflight checklist

- [ ] Handled rate limiting
- [ ] Handled pagination
- [ ] Implemented the code in async
- [ ] Support Multi account

## Screenshots

Include screenshots from your environment showing how the resources of the integration will look.

## API Documentation

Provide links to the API documentation used for this integration.

### CODE REVIEW

File: port_ocean/cli/cookiecutter/cookiecutter.json [hunk 1]
```diff
@@ -6,6 +6,9 @@
     "email": "Your address email <you@example.com>",
     "release_date": "{% now 'local' %}",
     "is_private_integration": true,
+    "port_client_id": "you can find it using: https://docs.getport.io/build-your-software-catalog/custom-integration/api/#find-your-port-credentials",
```
Comment (Tankilevitch) [inline port_ocean/cli/cookiecutter/cookiecutter.json:9]: I wonder if we can do something like this - https://cli.r-lib.org/reference/links.html

File: docs/framework-guides/docs/getting-started/getting-started.md [hunk 2]
```diff
@@ -44,18 +44,25 @@ $ straw@hat.com
...
+$ <your-port-client-secret>
 
-⚓️ Install necessary packages: Run make install to install all required packages for your project.
-▶️ cd ./my_integration && make install && . .venv/bin/activate
+is_us_region [y/n] (n):
+$ y
 
-⚓️ Set sail with Ocean: Run ocean sail <path_to_integration> to run the project using Ocean.
-▶️ ocean sail ./my_integration
+🌊 Ahoy, Captain! Your project is ready to set sail into the vast ocean of possibilities!
```
Comment (Tankilevitch) [inline docs/framework-guides/docs/getting-started/getting-started.md:59]: Love that 🤩

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
Very very nice, lets bump a version in pyproject.toml and add a changelog

File: port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example [hunk 3]
```diff
@@ -1,2 +1,6 @@
-OCEAN__PORT__CLIENT_ID="<port-client-id>"
-OCEAN__PORT__CLIENT_SECRET="<port-client-secret>"
+OCEAN__PORT__CLIENT_ID={{ cookiecutter.port_client_id }}
+OCEAN__PORT__CLIENT_SECRET={{ cookiecutter.port_client_secret }}
+OCEAN__INTEGRATION__IDENTIFIER={{ cookiecutter.integration_slug }}
+OCEAN__PORT__BASE_URL={% if cookiecutter.is_us_region %}https://api.us.getport.io{% else %}https://api.getport.io{% endif %}
+OCEAN__EVENT_LISTENER__TYPE=POLLING
```
Comment (matan84) [inline port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example:5]:
```suggestion
OCEAN__EVENT_LISTENER={"type": "POLLING"}
```

File: port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example [hunk 4]
```diff
@@ -1,2 +1,6 @@
-OCEAN__PORT__CLIENT_ID="<port-client-id>"
-OCEAN__PORT__CLIENT_SECRET="<port-client-secret>"
+OCEAN__PORT__CLIENT_ID={{ cookiecutter.port_client_id }}
+OCEAN__PORT__CLIENT_SECRET={{ cookiecutter.port_client_secret }}
+OCEAN__INTEGRATION__IDENTIFIER={{ cookiecutter.integration_slug }}
```
Comment (matan84) [inline port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example:3]:
```suggestion
OCEAN__INTEGRATION={"type": "{{ cookiecutter.integration_slug }}", "identifier": "my-{{ cookiecutter.integration_slug }}-integration", "config":{}}
```

File: port_ocean/cli/cookiecutter/cookiecutter.json (hunk 1 above)
Comment (yaelibarg) [reply port_ocean/cli/cookiecutter/cookiecutter.json:9]: i will look into it right now

**Review by matan84 — CHANGES_REQUESTED** [review]
Nice job!

File: port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example (hunk 3 above)
Comment (yaelibarg) [reply port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example:5]: why do you think we need this changed? if i change it, the .env file generated will be different

Comment (yaelibarg) [reply port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example]: decided to leave as is

File: port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example (hunk 4 above)
Comment (yaelibarg) [reply port_ocean/cli/cookiecutter/{{cookiecutter.integration_slug}}/.env.example]: decided to leave as is

File: port_ocean/cli/cookiecutter/cookiecutter.json (hunk 1 above)
Comment (yaelibarg) [reply port_ocean/cli/cookiecutter/cookiecutter.json:9]: we decided to leave it as is, since terminal does not support markdown

**Review by Tankilevitch — APPROVED** [review]
LGTM

**Review by matan84 — APPROVED** [review]
LGTM

**Pull request merged**